jumpscale/servers/gedis_http/__init__.py

```diff
@@ -11,12 +11,16 @@
 class GedisHTTPServer(Base):
     host = fields.String(default="127.0.0.1")
     port = fields.Integer(default=8000)
+    allow_cors = fields.Boolean(default=True)
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self._app = Bottle()
         self._client = None
-        self._app.route("/<package>/<actor>/<method>", ["GET", "POST"], self.handler)
+        http_methods = ["GET", "POST"]
```

Comment:
Can we add POST and DELETE to reflect what's returned by options header?

Reply:
updated in https://github.com/threefoldtech/js-ng/pull/529/files#diff-210a4f39e875b4b2069000b9e8ced9ce89092546503a4dfdadabbfef85df0fb3R22